finance/proposal-calls/Moore_2021.md

```diff
@@ -0,0 +1,90 @@
+# Call for Astropy ideas / projects to be funded in 2021
+
+## Goals
+Projects and ideas funded through this process shall improve the Astropy Project. This includes code for the core repository, for coordinated or affiliated packages, infrastructure (e.g. testing, CI), tutorials, workshops, notebooks, community engagement, inclusion, diversity and empowerment, travel to conferences or meetings, talks, mentoring, and anything else broadly Project related. This does not have to be a new initiative, but can be ongoing work (e.g. maintain the CI), but cannot include work that is already otherwise paid for (e.g. if you have a NASA grant to maintain the CI, you cannot bill for the same work twice).
+
+Suggested projects shall foster the Astropy Project goals of community, shared work, and cooperation and shall be proposed and executed in this manner.
+
+Total funding for this call is coming from the Moore grant and 200,000 $ are available in total in the period spring 2021 to spring 2022.
+
+Project suggetions need to clearly identify what the funding is used for (e.g. "pay YYY $ to add Klingon Standard Time to astropy.time" or "organize astropy workshop to engage amateur astronomer community in Antarctica")
+
+*Note: You can think of this as an internal "proposal process" to allow the community to select exactly where the money that the Astropy Project has will be spend. To avoid confusion with the external proposal that gave the money to Astropy (here: Moore foundation), we use the term "suggested project" and "project team" here.*
+
+## Details
+
+- Source of money: Moore grant
+- Specific area: Most of the money can be distributed to any area deemed important by the community. However, we do have specific budgets for work related to spectroscopy (22500 US $), mentoring contributors (15000 US $), and first contributor workshops (12000 US $).
+- Restrictions: This is a world-wide call, open to all current and future contributors to the Astropy project. (The only restriction is that money cannot be transferred to countries that the United States does not allow banking transactions with, e.g. North Korea).
+
+## Process
+### Suggestion
+Project ideas should be posted as pull requests to github.com/astropy/astropy-project/ with the initial title text "Funding 2021:".
+The goal is to make this process as easy as possible on everyone, so we impose a limit of 100 lines assuming 80 char/line. We provide a template below.
+We envision a two stage process to enable the community to comment and improve ideas, teams with similar ideas can merge to avoid duplicated work, etc..
+
+- 2021-03-20: Draft Deadline (after this date, no new projects)
```

Comment:
We need to make it clear somehow that comments and what not can start as soon as the PR is submitted. So the discussion period doesn't start March 21, it can start much sooner. 

```suggestion
- Discussion can start as soon as PRs are posted
- 2021-03-20: Draft Deadline (after this date, no new projects)
```